src/component/Timer/index.jsx

```diff
@@ -13,6 +13,7 @@ const Timer = () => {
   const [[mins, secs], setTime] = useState([INITIAL_PERIOD.mins, INITIAL_PERIOD.secs]);
   const [isRunning, setIsRunning] = useState(false);
   const [currentPeriod, setCurrentPeriod] = useState(INITIAL_PERIOD);
+  const [periodText, setPeriodText] = useState(INITIAL_PERIOD.id);
```

Comment:
You don't need to store the `periodText` in a separate state. You already have it available to you in the `currentPeriod`. On line 53, you can directly do:


```jsx
<h3>{currentPeriod.id}</h3>
```

And that way you won't need this new state.